src/main/java/me/arminb/sara/entities/User.java

```diff
@@ -0,0 +1,45 @@
+package me.arminb.sara.entities;
+
+
+import com.sun.istack.internal.NotNull;
+import org.bson.types.ObjectId;
+
+//import javax.validation.constraints.Size;
+
+public class User {
+
+    private ObjectId _id;
+
+    private String user_name;
```

Comment:
why not username, email and password instead of user_password, user_email and user_name?